modules/ROOT/pages/get-mapping-recommendations.adoc

```diff
@@ -19,13 +19,13 @@ This view shows the expected structures of the input and output messages. Mappin
 
 image::simple-transform-card-example.png[]
 
-If you are not sure how to map an input payload to an output payload, you can have the Transform card recommend mappings between them.
+If you are not sure how to map an input data type to an output type, you can have the Transform card recommend mappings between them.
 
 == Limitations
 
 * The card does not make recommendations for mapping input attributes to output attributes or input variables to output variables.
 
-* Elements of an array that is itself an element in an array are ignored in the recommendations. For example, both d and e in this payload are ignored:
+* Elements of an array that is itself an element in an array are ignored in the recommendations. For example, both d and e in this data type are ignored:
```

Comment:
`d` and `e`  ?

Reply:
Those are keys in an example of JSON that follows the sentence.

Reply:
Ah, but they do need tick marks.